packages/framework/src/Form/Constraints/NotNegativeMoneyAmount.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class NotNegativeMoneyAmount extends Constraint
```

Comment:
This constraint will be used in `\Shopsys\FrameworkBundle\Form\Admin\TransportAndPayment\FreeTransportAndPaymentPriceLimitsFormType` where it constrains the price limit, not the price itself. Also, it may be used for other general money amounts - amount of VAT paid, a tip, etc.